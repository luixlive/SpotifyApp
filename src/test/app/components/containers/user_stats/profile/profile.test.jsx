import _ from 'lodash';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';
import React from 'react';
import renderer from 'react-test-renderer';

import initialState from './../../../../../test_utils/initial_state';
import Profile
  from './../../../../../../app/components/containers/user_stats/profile';

describe('App Components - Profile', () => {
  it('renders', () => {
    const mockStore = configureStore();
    const store = mockStore(_.merge({}, initialState, {
      user: { profile: { images: [{ url: 'url' }] } },
    }));
    const rendered = renderer.create((
      <Provider store={store}>
        <Profile />
      </Provider>
    )).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
