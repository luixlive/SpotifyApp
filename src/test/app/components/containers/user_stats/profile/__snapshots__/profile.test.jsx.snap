// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Components - Profile renders 1`] = `
<div
  className="ui basic segment"
>
  <div
    className="ui three column grid"
  >
    <div
      className="four wide column"
    />
    <div
      className="center aligned eight wide column"
    >
      <a
        href=""
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="ui small circular centered image"
          onError={[Function]}
          src="url"
        />
      </a>
      <div
        className="ui fluid card"
        href={undefined}
        onClick={[Function]}
      >
        <div
          className="center aligned content"
        >
          <div
            className="header"
          >
            <h1
              className="ui header"
            >
              
            </h1>
          </div>
          <div
            className="meta"
          >
            <h3
              className="sub header"
            >
              
               (
              <a
                href=""
                rel="noopener noreferrer"
                target="_blank"
              >
                
              </a>
              )
            </h3>
          </div>
        </div>
        <div
          className="extra center aligned content"
        >
          0
           followers
        </div>
      </div>
    </div>
    <div
      className="four wide column"
    />
  </div>
</div>
`;
